File: Modul-4/Soal-2/PRAK402-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int batas, i;
    printf("Masukkan batas maksimal dari bilangan yang akan dihitung: ");
    scanf("%d", &batas);

    for (i = 1; i <= batas; i += 2) {
        printf("%d ", i);
    }
    printf("\n");

    for (i = (batas % 2 == 0 ? batas : batas - 1); i >= 2; i -= 2) {
        printf("%d ", i);
    }
    printf("\n");

    return 0;
}

File: Modul-4/Soal-2/PRAK402-2410817320001-NazlaSalsabila.py
batas = int(input("Masukkan batas maksimal dari bilangan yang akan dihitung: "))

for i in range(1, batas + 1, 2):
    print(i, end=" ")

print()

i = batas if batas % 2 == 0 else batas - 1
while i >= 2:
    print(i, end=" ")
    i -= 2

print()